data/examples/multicluster/istio-gateway.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  hub: gcr.io/istio-testing
+  tag: 1.4-dev
+  defaultNamespace: istio-system
+  trafficManagement:
+    enabled: false
+
+  policy:
+    enabled: false
+
+  telemetry:
+    enabled: false
+
+  security:
+    enabled: false
```

Comment:
Citadel is required for mTLS.

Reply:
This was actually totally wrong due to too many tasks going on in parallel. 
PTAL at the updated values which should correspond to directions here:
https://preliminary.istio.io/docs/setup/install/multicluster/gateways/#deploy-the-istio-control-plane-in-each-cluster
and here:
https://preliminary.istio.io/docs/setup/install/multicluster/shared-gateways/#setup-cluster-1-primary